packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -113,6 +120,26 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether a drag handle is shown when [enableDrag] is true.
```

Comment:
Responded to https://github.com/flutter/flutter/pull/120855#issuecomment-1464735424 on that PR, but think the drag handle should be shown if this is true independently from `enableDrag`, whose docs can be clarified to say this applies to the content below the handle, if `showDragHandle` is true.